src/Component/ModalTask/ModalTask.jsx

```diff
@@ -13,9 +38,25 @@ const ModalTask = ({ setModalTaskFlag }) => {
           <div className="flex  items-end sm:items-center justify-center min-h-full p-4 text-center sm:p-0">
             <div className="relative bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all sm:my-8 sm:max-w-lg sm:w-full">
               <div className="bg-white px-4 pt-5 pb-4 sm:p-6 sm:pb-4">
-                <form className="">
+                <form
+                  onSubmit={(event) => {
+                    event.preventDefault();
+                    taskDispatch({
+                      type: ACTION_TYPE.SUMBIT_TASK,
+                      payload: taskDetails,
+                    });
+                    notify(message);
+                    setModalTaskFlag((flag) => !flag);
+                  }}
```

Comment:
You can extract this function for more clean code. Suggestion!

Reply:
sure .